### PR TITLE
corrected translation to english errors in 'var' lesson

### PR DESCRIPTION
**Pull request by tgentry300**

### CODE REVIEW

**Comment by iliakan** [comment]
Are you a native speaker?

**Comment by davegregg** [comment]
Yes, @iliakan, he's one of my students.

File: 1-js/06-advanced-functions/04-var/article.md [hunk 1]
```diff
@@ -169,16 +169,16 @@ function sayHi() {
 sayHi();
 ```
 
-Because all `var` declarations are processed at the function start, we can reference them at any place. But variables are undefined until the assignments.
+Because all `var` declarations are processed at the function start, we can reference them at any place. But variables are undefined until they are explicitly defined.
```
Comment (iliakan) [inline 1-js/06-advanced-functions/04-var/article.md]: Undefined until they are explicitly assigned (=).

File: 1-js/06-advanced-functions/04-var/article.md [hunk 2]
```diff
@@ -169,16 +169,16 @@ function sayHi() {
...
 
 In both examples above `alert` runs without an error, because the variable `phrase` exists. But its value is not yet assigned, so it shows `undefined`.
 
 ## Summary
 
 There are two main differences of `var`:
 
-1. Variables have no block scope, they are visible minimum at the function level.
-2. Variable declarations are processed at function start.
+1. `var` Variables have no block scope, they are not block-local and can be accessed from any point in the script.
```
Comment (iliakan) [inline 1-js/06-advanced-functions/04-var/article.md]: wrong, they are function-local, so can't be "accessed from any point in the script"  =)

Comment (iliakan) [reply 1-js/06-advanced-functions/04-var/article.md]: not sure if rephrasing is good here...

**Comment by iliakan** [comment]
Let's ensure maximum clarity and correctness with this subtle topic.

File: 1-js/06-advanced-functions/04-var/article.md (hunk 2 above)
Comment (iliakan) [reply 1-js/06-advanced-functions/04-var/article.md]: LGTM

File: 1-js/06-advanced-functions/04-var/article.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ In the very first chapter about [variables](info:variables), we mentioned three
 
 But `var` is a very different beast, that originates from very old times. It's generally not used in modern scripts, but still lurks in the old ones.
 
-If you don't plan meeting such scripts you may even skip this chapter or postpone it, but then there's a chance that it bites you later.
+If you don't plan on reading or editing such scripts you may even skip this chapter, but there is a chance that you will see `var` in old scripts that you are reading and/or editing.
```
Comment (iliakan) [inline 1-js/06-advanced-functions/04-var/article.md]: Double wording: `reading/editing`.

**Pull request closed without merging**